app/logging/raw_db_logger_test.go

```diff
@@ -0,0 +1,123 @@
+package logging
+
+import (
+	"errors"
+	"testing"
+
+	"bou.ke/monkey"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/France-ioi/AlgoreaBackend/app/config"
+	"github.com/France-ioi/AlgoreaBackend/app/loggingtest"
+)
+
+func TestNewRawDBLogger_ErrorFallback(t *testing.T) {
+	patch := monkey.Patch(config.Load, func() (*config.Root, error) {
```

Comment:
I still don't agree on monkey patching an internal lib on which we have control. 🤨 Or is a temporary solution while the other PR is not complete? Why don't you just create the db with your hand-made config? Looks much cleaner.

Reply:
`TestNewDBLogger_ErrorFallback` does exactly the same (see here https://github.com/France-ioi/AlgoreaBackend/blame/master/app/logging/dbLogger_test.go#L21). We need to check error handling and I don't see another way to do this.